### ISSUE TITLE
Improve traffic behavior

### ISSUE THREAD

**Issue by mole99** [rwengine/openrw#554]
Currently the driving behavior in OpenRW is only rudimentary implemented.

Tasks which should be tackled in the future are:

- [ ] Implement dead ends
- [ ] Implement one-way streets (problem: in our node system we have
 no possibility to obtain the direction of a street)
- [ ] Add areas where it is illegal to drive (e.g. destroyed bridge at the start of the game) 
- [ ] Vehicles sometimes lock each other (maybe add an unstuck activity)
- [ ] Implement different driving styles (e.g. rampant, fleeing...)
- [ ] Implement opcodes (opcode 00ae was partially implemented)

**Comment by ShFil119** [comment]
Traffic director ignores debug menu and vehicles don't use "smooth steering".

Nice feature would be" another color for line when something is blocking the path. (Or dotted line)

**Comment by mole99** [comment]
Just to be clear, while in debug menu the vehicles should not move, right?

**Comment by ShFil119** [comment]
Yeah, controllers and all physics should be stopped. It looks like for physics should be created additional issue. Hmm...
